docs/shared/recipes/running-tasks/customizing-outputs.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Cache Results with Outputs
+description: 'Learn how to optimize what Nx restores from the cache'
+---
+
+# Cache Results with Outputs
+
+Whenever Nx runs a task which can be cached, it will store the results of that task in the cache.
+If nothing has changed, when Nx runs the task again, it will restore the results from the cache instead of spending the time to run the task again.
```

Comment:
```suggestion
When Nx runs the task again, if nothing has changed, it will restore the results from the cache instead of spending the time to run the task again.
```